src/stores/index.js

```diff
@@ -1020,6 +1031,76 @@ class Store {
     })
   }
 
+  getLeaderBoard = (_payload) => {
+    
+    const account = store.getStore('account')
+    const web3 = new Web3(store.getStore('web3context').library.provider);
+    const memes = this.getMemes();
+
+    async.map(memes, (meme, callback) => {
+        this._getLeaderBoardData(web3, meme, callback);
+    }, async(err, leaderboardData) => {
+      if(err) {
+        return emitter.emit(ERROR, err);
+      }
+      console.log(leaderboardData);
```

Comment:
can you please remove all the console logs in your PR? they look distracting and exposing a lot of info to strangers

Reply:
It was put there for the Shaftoe to verify the data

Reply:
It was put there for the Shaftoe to verify the dataI will

Reply:
I will remove all comments once Shaftoe has verified everything